isis/tests/Fixtures.h

```diff
@@ -3,430 +3,21 @@
 
 #include "gtest/gtest.h"
 
-#include <memory>
-#include <string>
-
 #include <QString>
 #include <QTemporaryDir>
-#include <QTemporaryFile>
-
-#include <nlohmann/json.hpp>
-
-#include "csm/csm.h"
-#include "csm/Ellipsoid.h"
+#include <QString>
 
-#include "Cube.h"
-#include "IException.h"
-#include "OriginalLabel.h"
-#include "Pvl.h"
-#include "PvlGroup.h"
-#include "PvlObject.h"
-#include "ImagePolygon.h"
-#include "PolygonTools.h"
 #include "Blob.h"
-#include "MockCsmPlugin.h"
-#include "Mocks.h"
-#include "ControlNet.h"
-#include "FileList.h"
-#include "FileName.h"
-
-#include <geos/io/WKTReader.h>
-#include <geos/io/WKTWriter.h>
-#include "geos/geom/CoordinateArraySequence.h"
-#include "geos/geom/CoordinateSequence.h"
-#include "geos/geom/LinearRing.h"
-#include "geos/geom/Polygon.h"
+#include "Pvl.h"
 
-using json = nlohmann::json;
+#include "TempFixtures.h"
+#include "CameraFixtures.h"
+#include "CsmFixtures.h"
+#include "CubeFixtures.h"
+#include "NetworkFixtures.h"
```

Comment:
I left file here for two reasons:

1. If anyone adds new fixtures in an existing PR, this file will still be here and they won't need to deal with merge conflicts
2. You can still just include all of the fixtures by including this file because it includes the split up files now.